dev/workflow/rocoto/gfs_tasks.py

```diff
@@ -680,12 +680,20 @@ def esnowanl(self):
     def prepoceanobs(self):
 
         ocean_hist_path = self._template_to_rocoto_cycstring(self._base["COM_OCEAN_HISTORY_TMPL"], {'RUN': 'gdas'})
+        self._prepoceanobs = self.app_config.configs[next(iter(self.app_config.configs))]['prepoceanobs']
+        dmpdir = self._prepoceanobs["DMPDIR"]
```

Comment:
```suggestion
        dmpdir = self._configs['prepoceanobs']["DMPDIR"]
```
`DMPDIR` is a known key in `config.prepoceanobs` https://github.com/NOAA-EMC/global-workflow/blob/f9afc6144e6056b873dc7468f861d0cfe5e6645e/dev/parm/config/gfs/config.prepoceanobs.j2#L19.  Is there a need to "search" for this config, when it is known.